### PR TITLE
docs only: fix spelling error

### PR DESCRIPTION
**Pull request by wbpcode**
Commit Message: docs only: fix spelling error
Additional Description:

Fix the spelling error in previous PR #23157.

Risk Level: n/a.
Testing: n/a.
Docs Changes: n/a.
Release Notes: n/a.
Platform Specific Features: n/a.

### CODE REVIEW

**Comment by wbpcode** [comment]
cc @phlax

File: docs/root/api-v3/config/filter/generic/generic.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Dubbo filters
+Generic filters
 ==============
```
Comment (phlax) [inline docs/root/api-v3/config/filter/generic/generic.rst:2]: you will need to make the unerline match the title length

Comment (wbpcode) [reply docs/root/api-v3/config/filter/generic/generic.rst]: done.

**Review by phlax — APPROVED** [review]
lgtm, thanks @wbpcode

**Pull request merged**